src/test/java/io/stargate/sgv2/jsonapi/api/v1/CreateCollectionIntegrationTest.java

```diff
@@ -396,6 +411,70 @@ public void duplicateVectorCollectionNameWithDiffSetting() {
           .statusCode(200)
           .body("status.ok", is(1));
     }
+
+    @Test
+    public void failWithInvalidNameInIndexingAllows() {
+      // create a vector collection
+      given()
+          .header(HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME, getAuthToken())
+          .contentType(ContentType.JSON)
+          .body(
+              """
+                    {
+                      "createCollection": {
+                        "name": "collection_with_bad_allows",
+                        "options" : {
+                          "indexing" : {
+                            "allow" : ["field", "$eq"]
```

Comment:
Can we add a test field with array index fails? `address.[0].city` should throws error?

Reply:
Good idea, will add.